mindmeld/query_cache.py

```diff
@@ -68,11 +69,11 @@ def __init__(self, app_path):
         # Create table to store the data version
         cursor.execute("""
         CREATE TABLE IF NOT EXISTS version
-        (version_number INTEGER PRIMARY KEY);
+        (text_pipeline_hash TEXT PRIMARY KEY, config_id INTEGER);
```

Comment:
In light of the comment above I'd like to see this as simply `schema_version_hash TEXT PRIMARY KEY` and we can drop any other versioning schemes.

Reply:
Done!